### PR TITLE
[LIB-157] Add in_bulk get 

### PR DESCRIPTION
**Pull request by opalczynski**

### CODE REVIEW

File: tests/integration_test_batch.py [hunk 1]
```diff
@@ -101,3 +101,31 @@ def test_batch_mix(self):
...
+
+        for res_id, res in bulk_res.iteritems():
+            self.assertEqual(res_id, res.id)
+
+        self.assertEqual(bulk_res[self.update1.id].title, self.update1.title)
+        self.assertEqual(bulk_res[self.update2.id].title, self.update2.title)
+        self.assertEqual(bulk_res[self.update3.id].title, self.update3.title)
+
+        # test class bulk
+
```
Comment (Zhebr) [inline tests/integration_test_batch.py]: seems too sparse

**Comment by Zhebr** [comment]
not sure about name ```in_bulk```; however, looks good

**Comment by opalczynski** [comment]
@Zhebr 
Just following the django: https://docs.djangoproject.com/en/1.9/ref/models/querysets/#in-bulk

But if you have another proposition - I can re-think that :)

File: tests/integration_test_batch.py (hunk 1 above)
Comment (opalczynski) [inline tests/integration_test_batch.py]: What do you mean?

**Comment by dancio** [comment]
lgtm

**Comment by Zhebr** [comment]
@opalczynski hmm... let's follow Django way then

**Comment by 23doors** [comment]
lgtm

**Pull request merged**